### ISSUE TITLE
Scale of image

### ISSUE THREAD

**Issue by Iskander1220** [AdeelK93/wapo_pdf#1]
Hi, AdeelK93, and sorry for my english!

Do I understand correctly that on this link (https://github.com/AdeelK93/wapo_pdf/blob/823139583d49f48f7dadc650cd50266eba1a1f69/scales.R) is the calculation of the scale image value? Could you explain how this calculation is?

**Comment by AdeelK93** [comment]
This thing is super obfuscated. In the [pressreader javascript](https://r.prcdn.co/scripts/se2sky/2.40.484/viewer.nolibs.build.min.js?callback=loadCallback2), there is this function, from where I take that calculation:
```js
calculateMagnifierScale: function(e) {
                        var t = l.magnifierPageSizes()[e];
                        return t ? Math.floor(Math.min(100 * t.W / this.width(), 100 * t.H / this.height())) : 0
                    },
```
That `l.magnifierPageSizes` comes from an endpoint at [this address](https://svc.pressreader.com/se2skyservices/IssueInfo/GetIssueInfoByCid) that only ever returns the widths and heights `1728` and `3024`.
Rather than scraping that endpoint every time, I instead hardcoded those values and this calculation. Does that help?

**Comment by Iskander1220** [comment]
Do I understand correctly - on the link `GetIssueInfoByCid` we getting the highest values `W` and `H`? And then what principle do calculations occur?

**Comment by AdeelK93** [comment]
Yeah that's the highest resolution available. What do you mean about `what principle`?

**Comment by Iskander1220** [comment]
There was meant which computation there are produced. Is it possible to determine which values ​​are taken [from link](https://ingress.pressreader.com/services/IssueInfo/GetIssueInfoByCid?accessToken=GAg2B723fH-5YkIHgHLd2NtiuOBhyKwGmCYliMmQGfzDUqXpTDMFzAxysiUfelPgUSd_tK8Jf8DcfYqBA4CUHw!!&cid=0004), and how are they multiplied?
